client/src/components/prospects/Checkbox.jsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import {Checkbox as MaterialCheckbox} from "@material-ui/core";
+
+export default function Checkbox(props) {
+  const [checked, setChecked] = React.useState(props.checked);
+
+  const handleChange = (event) => {
+    setChecked(event.target.checked);
+
+    // Make sure the clickEvent prop was set first.
+    if (props['clickEvent']){
```

Comment:
```suggestion
    if (props.clickEvent){
```

I would access properties like this just so its consistent with others as well